#include <exception>
#include "../utils/test_utils.hpp"

int main()
{
    TEST_ASSERT(true);

    return 0;
}